app/scripts/directives/resources.js

```diff
@@ -1,6 +1,65 @@
 'use strict';
 
 angular.module('openshiftConsole')
+  .directive('containerStatuses', function($filter) {
+    return {
+      restrict: 'E',
+      scope: {
+        pod: '=',
+        onDebugTerminal: '=?',
+        detailed: '=?'
+      },
+      templateUrl: 'views/_container-statuses.html',
+      link: function(scope) {
+        scope.hasDebugTerminal = angular.isFunction(scope.onDebugTerminal);
+
+        var isContainerTerminatedSuccessfully = $filter('isContainerTerminatedSuccessfully');
+        var haveAllContainersTerminatedSuccessfully = function(containerStatuses) {
+          return _.every(containerStatuses, isContainerTerminatedSuccessfully);
+        };
+
+        scope.$watch('pod', function(updatedPod) {
+          scope.initContainersTerminated = haveAllContainersTerminatedSuccessfully(updatedPod.status.initContainerStatuses);
+
+          if (scope.expandInitContainers !== false) {
+            scope.expandInitContainers = !scope.initContainersTerminated;
+          }
+        });
+
+        scope.toggleInitContainer = function() {
+          scope.expandInitContainers = !scope.expandInitContainers;
+        };
+
+        scope.showDebugAction = function (containerStatus) {
+
+          if (_.get(scope.containerStatuses, 'status.phase') === 'Completed') {
```

Comment:
Can do, when updated the scope property the old search and replace missed a couple